README.md

```diff
@@ -58,20 +30,15 @@ After you're finished please remove all the comments and instructions!
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Hello Microverse] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[Hello Microverse]** is a...
```

Comment:
- [ OPTIONAL] I think if you can complete the description of your project that would be a good idea because it helps the visitors understand what your project is about. 👌 😊